pkgs/applications/science/electronics/kicad/unstable.nix

```diff
@@ -0,0 +1,52 @@
+{ config, wxGTK, lib, pkgs, stdenv, fetchFromGitHub, cmake, mesa, zlib
```

Comment:
- `config` is not used
- `pkgs` should be removed

Reply:
Done locally.
Should I squash it in the previous commit to update the PR, or add a new commit?